canal/canal_test.go

```diff
@@ -133,101 +132,3 @@ func (s *canalTestSuite) TestCanalFilter(c *C) {
 	c.Assert(errors.Cause(err), Equals, ErrExcludedTable)
 	c.Assert(sch, IsNil)
 }
-
-func TestCreateTableExp(t *testing.T) {
```

Comment:
it's better to keep these unit test, and modify them to suit new codes

Reply:
i think is unnecessary, new codes  of parse  had  all been test in parser pkg

Reply:
you should make sure it's what you need, not just want 🤔